Cargo.toml

```diff
@@ -25,6 +25,7 @@ version = "0.6.0"
 
 [package.metadata.docs.rs]
 features = ["stm32l0x2", "rt", "stm32-usbd"]
```

Comment:
Slightly offtopic, but you should probably include an `io` feature (e.g. `io-STM32L071`) as well, otherwise - if I understand this correctly - the `serial` and `i2c` modules will be missing from the docs.